src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -267,6 +267,21 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,    soeREQUIRED}
         },
         commonFields);
+
+    add(jss::CFTokenIssuance,
+        ltCFTOKEN_ISSUANCE,
+        {
+            {sfIssuer,               soeREQUIRED},
+            {sfTransferFee,          soeDEFAULT},
+            {sfOwnerNode,            soeREQUIRED},
+            {sfAssetCode,            soeREQUIRED},
+            {sfAssetScale,           soeDEFAULT},
+            {sfMaximumAmount,        soeOPTIONAL},
+            {sfOutstandingAmount,    soeREQUIRED},
+            {sfLockedAmount,         soeREQUIRED},
```

Comment:
The specs has `LockedAmount` as default.
Also the specs has `OutstandingAmount` as required in the table but default in the field description.

Reply:
This change will be part of another PR.